corellium/adapter/src/test/kotlin/flank/corellium/adapter/Example.kt

```diff
@@ -63,14 +64,14 @@ fun main() {
     runBlocking {
 
         println("* Authorizing")
-        api.authorize(credentials)
+        api.authorize(credentials).join()
 
         println("* Invoking devices")
-        val ids = api.invokeAndroidDevices(AndroidInstance.Config(2)).toMutableList()
+        val ids = api.invokeAndroidDevices(AndroidInstance.Config(2)).toList().toMutableList()
```

Comment:
.toList().toMutableList()
Is there a reason for this?
If this is already a collection its not needed.